emails/beneficiary_warning_after_pro_booking_cancellation.py

```diff
@@ -0,0 +1,44 @@
+from typing import Dict
+
+from models import Booking
+from repository.feature_queries import feature_send_mail_to_users_enabled
+from utils.date import ENGLISH_TO_FRENCH_MONTH
+from utils.mailing import SUPPORT_EMAIL_ADDRESS, DEV_EMAIL_ADDRESS, format_booking_hours_for_email, get_event_datetime
+
+
+def retrieve_data_to_warn_beneficiary_after_pro_booking_cancellation(booking: Booking) -> Dict:
+    is_event = booking.stock.beginningDatetime is not None
+    if is_event:
+        event_date = _format_booking_date(booking)
+        event_hour = format_booking_hours_for_email(booking)
+    else:
+        event_date = ''
+        event_hour = ''
+
+    return {
+        'FromEmail': SUPPORT_EMAIL_ADDRESS if feature_send_mail_to_users_enabled() else DEV_EMAIL_ADDRESS,
```

Comment:
```suggestion
        'FromEmail': SUPPORT_EMAIL_ADDRESS,,
```
Par conséquent, tu peux retirer des tests.

Reply:
on les envoit toujours depuis support ?

Reply:
Oui, ça été discuté avec Nicolas.